dis_snek/models/discord_objects/channel.py

```diff
@@ -164,31 +164,38 @@ def history(
     async def get_messages(
         self,
         limit: int = 50,
-        around: "Snowflake_Type" = MISSING,
-        before: "Snowflake_Type" = MISSING,
-        after: "Snowflake_Type" = MISSING,
+        around: Union["Timestamp", "Snowflake_Type"] = MISSING,
+        before: Union["Timestamp", "Snowflake_Type"] = MISSING,
+        after: Union["Timestamp", "Snowflake_Type"] = MISSING,
     ) -> List["Message"]:
         """
         Fetch multiple messages from the channel.
 
         Args:
-            limit: Max number of messages to return, default `50`, max `100`
-            around: Message to get messages around
-            before: Message to get messages before
-            after: Message to get messages after
+            limit: Max number of messages to return, default `50`
+            around: Snowflake | Timestamp to get messages around
+            before: Snowflake | Timestamp to get messages before
+            after: Snowflake | Timestamp to get messages after
 
         Returns:
             A list of messages fetched.
         """
-        if limit > 100:
-            raise ValueError("You cannot fetch more than 100 messages at once.")
 
         if around:
-            around = to_snowflake(around)
-        elif before:
-            before = to_snowflake(before)
-        elif after:
-            after = to_snowflake(after)
+            if type(around) == Timestamp:
+                around = around.to_snowflake()
+            else:
+                around = to_snowflake(around)
+        if before:
+            if type(before) == Timestamp:
+                before = before.to_snowflake()
+            else:
+                before = to_snowflake(before)
+        if after:
+            if type(after) == Timestamp:
+                after = after.to_snowflake()
+            else:
+                after = to_snowflake(after)
```

Comment:
That's  a lot of code duplication
I would suggest making a helper function to perform this conversion, or, preferably, including "snowflake from timestamp" in `to_snowflake` function itself